frontend/src/lib-common/finite-date-range.ts

```diff
@@ -123,6 +124,13 @@ export default class FiniteDateRange {
     return this.start.isEqual(other.start) && this.end.isEqual(other.end)
   }
 
+  getGaps(childRanges: FiniteDateRange[]): FiniteDateRange[] {
+    const gapDates = [...this.dates()].filter(
+      (date) => childRanges.find((range) => range.includes(date)) === undefined
```

Comment:
I think using `!childRanges.some((range) => range.includes(date))` or `childRanges.every((range) => !range.includes(date))` makes the intent clearer here.